### ISSUE TITLE
Update readme (remove link to datavizta)

### ISSUE THREAD

**Issue by demeringo** [Boavizta/environmental-footprint-data#95]
As we do display  anymore this dataset in Datavizta (https://dataviz.boavizta.org/), we should remove the link that mention it in the readme of this repository.  

This may lead to confusion (people may believe that this data set is the same thing as the Boavizta API).

See https://github.com/Boavizta/environmental-footprint-data/blob/61323eb5c3bb63b0703e7a9c6456482ccbaac3a6/README.md?plain=1#L11

Is the dataset in this repository really not maintained anymore  (as explained by the disclaimer in Datavizta) ?

- if yes, we should also reflect it in the readme of _this_ repository and be explicit that we do not expect contributions anymore (and maybe also update the github project description). See https://github.com/Boavizta/environmental-footprint-data/blob/61323eb5c3bb63b0703e7a9c6456482ccbaac3a6/README.md?plain=1#L30
- otherwise, we need to update the disclaimer in Datavizta.

**Comment by demeringo** [comment]
@AirLoren what is your view on this ?

**Comment by AirLoren** [comment]
I agree this lay lead to confusion and I think we should delete this repository as we know many people using this dataset to evaluate impacts  even if we said that it is not relevant to perform such type of activities.
